### DISCUSSION TITLE
GitHub Education Student package location error

### DISCUSSION THREAD

**Discussion by whoIsClownHere** [community/community#76747] [GitHub Education]
### Hi everyone,

I'm trying to get a GitHub Education Student package, but I get this message. In my Safari browser, I have enabled the location for the website, but it doesn't work. What should I do in this situation?

When you click "Continue", you will be prompted to inform us of your location. Specifying your current location will help us confirm your affiliation with the school you have chosen.

To proceed, you must change your browser settings to allow GitHub to use your location information.

**Comment by BenTomsett** [comment]
I'm also having this issue with both Google Chrome and Safari (latest versions) on macOS Sonoma

**Comment by HiWebDev0026** [comment]
To resolve the issue with enabling location settings for the GitHub Education Student package in Safari, follow these steps:

Check Browser Settings:

Open Safari and go to the GitHub website.
Click on the "Safari" menu and select "Preferences."
Navigate to the "Websites" tab and then select "Location" from the sidebar.
Ensure that GitHub is not listed under the "Deny" or "Ask" columns. If it is, click on the dropdown menu next to it and select "Allow."
Clear Safari Cache and Cookies:

Sometimes, browser cache and cookies can cause issues with website functionalities. Clearing them might help in this case.
Try Another Browser:

If the issue persists, try using a different browser such as Google Chrome or Mozilla Firefox to see if the location settings prompt works correctly there.
Contact GitHub Support:

If the issue persists despite trying the above steps, consider reaching out to GitHub support for further assistance. They may have specific troubleshooting steps for Safari users.
By following these steps, you should be able to resolve the issue with enabling location settings for the GitHub Education Student package.

**Reply by whoIsClownHere** [reply]
<img width="771" alt="Screenshot 2023-12-09 at 21 48 45" src="https://github.com/community/community/assets/106624742/98611278-c0a1-4702-a412-d41bf2afb1d3">

<img width="551" alt="Screenshot 2023-12-09 at 21 49 21" src="https://github.com/community/community/assets/106624742/908f6f77-9e22-4e5d-9691-0bbf33380ad6">

I did what you said, but for some reason I still have this message.

**Comment by azidanimoh** [comment]
I'm also having this issue Google Chrome and Brave

**Comment by SouceCalve** [comment]
Also, but on Firefox

**Comment by Prerrna** [comment]
facing the same issue

**Comment by kevalmehta17** [comment]
Same Issue

**Comment by sri113-collab** [comment]
Any update here?

**Comment by CharlesH-max** [comment]
any update?

**Comment by FlyCastt** [comment]
I was facing the GitHub Education Student package location error #76747 and after some troubleshooting, I realized the issue was related to an incorrect configuration in the local settings. It’s always a good idea to check the system requirements and ensure all dependencies are correctly installed. If you’re looking for a better way to manage and optimize your coding projects, you should definitely check out [FlyCast](http://flycasts.net/). It’s a great platform to streamline your workflow!

**Comment by oleteacher** [comment]
Sadly, this is still happening and cannot reverify. Have checked many settings and still facing issue. Looks like I will have to say goodbye to GitHub education. It is happening to many at school district, we all were able to verify year or so back, but no longers. Sad to see it go.
